zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -6011,6 +6042,37 @@ components:
         processDefinitionName:
           type: string
           description: The name of the process definition (fall backs to the process definition ID if not available).
+    ProcessInstanceSequenceFlowsQueryResult:
+      description: Process instance sequence flows query response.
+      type: object
+      properties:
+        items:
+          description: The sequence flows.
+          type: array
+          items:
+            $ref: "#/components/schemas/ProcessInstanceSequenceFlowResult"
+    ProcessInstanceSequenceFlowResult:
+      description: Process instance sequence flows result.
+      type: object
+      properties:
+        sequenceFlowId:
+          type: string
+          description: The sequence flow ID.
```

Comment:
💭 Is this really only the sequence flow ID from the model? It is the sequence flow key, right? It's an internal identifier, unique across process instances since it contains the process instance key as a prefix as well, isn't it? It's not the user-provided ID from the model alone.

Reply:
Yes, it is just the generated ID with `processInstanceKey_flowNodeId`. I think I will just drop it from the response, it doesn't provide any value I think.

Reply:
Yeah, I was wondering if you could do anything with it but I guess you can't 🤔